src/Codeception/Lib/Driver/Db.php

```diff
@@ -288,4 +288,22 @@ protected function flushPrimaryColumnCache()
 
         return empty($this->primaryKeys);
     }
+    
+    public function update($table, array $data, array $criteria)
+    {
+        if (empty($data)) {
+            throw new \Exception(
```

Comment:
A more specific exception would be better. Codeception ModuleException is a good choice IMO.